src/candela/VisComponent/mixin/AutoResize.js

```diff
@@ -0,0 +1,16 @@
+import Resize from './Resize';
+
+let AutoResize = Base => class extends Resize(Base) {
```

Comment:
I feel like AutoResize should extend InitSize, since I can't think of where you would want AutoResize and not AutoSize.